web/web/src/app/rootLayout/AppBar.js

```diff
@@ -56,14 +66,31 @@ const AppBar = () => {
   const [metalakes, setMetalakes] = useState([])
   const router = useRouter()
   const logoSrc = (process.env.NEXT_PUBLIC_BASE_PATH ?? '') + '/icons/gravitino.svg'
+  const username = 'apache'
+  const repository = 'gravitino'
+  const [githubData, setGithubData] = useState({ stars: 0, forks: 0 })
 
   useEffect(() => {
+    const fetchGitHubData = async () => {
+      try {
+        const response = await axios.get(`https://api.github.com/repos/${username}/${repository}`)
+        let { stargazers_count, forks_count } = response.data
```

Comment:
In gravitino, axios have imported at `gravitino/web/web/src/lib/utils/axios/index.js` and export `defHttp`,.
And all the response data are all managed. 
I think the github star data can managed at `/gravitino/web/web/src/lib/store/sys/index.js`

Let me know if you have any questions.

Reply:
Hi @LauraXia123,
Thanks for suggestion.

I have some question for you.

1. The format filter function that you mean is like image below?
![image](https://github.com/user-attachments/assets/37858aa1-6a61-4883-bb22-c6b9e1810b40)

2. How to use defHttp to get GitHub API, I use ```const response = defHttp.get({url: githubAIP})```, but always response AxiosError, what's wrong with my code?
![image](https://github.com/user-attachments/assets/0e3cfea0-e720-43b6-83d4-6738a9d92df6)

3. How can I get the svgs in ```/gravitino/web/web/src/lib/icons/svg/```
4. Why do you suggest me to add overrideSrc, I use this keyword to search information, but I can't find anything. what does this attribute means?

thx